src/preview.jsx

```diff
@@ -276,9 +303,6 @@ export default class Preview extends React.Component {
     const items = data.map((item, index) => this.getElement(item, index));
     return (
       <div className={classes}>
-        <div className="edit-form" ref={this.editForm}>
-          {this.props.editElement !== null && this.showEditForm()}
-        </div>
```

Comment:
I could see this being pulled into the main library if we allowed a flag to be passed into the main component,

(again, thinking ADD-ONLY changes, no removals)

```js
<ReactFormBuilder
    showInlineEditForm={true}
/>
```

then

```js
{!showInlineEditForm && {
        <div className="edit-form" ref={this.editForm}>
          {this.props.editElement !== null && this.showEditForm()}
        </div>
})
```

Reply:
fixed